rest/data/pod.go

```diff
@@ -42,3 +83,48 @@ func (c *MockPodConnector) CheckPodSecret(id, secret string) error {
 	}
 	return errors.New("pod does not exist")
 }
+
+func validatePod(p restModel.APICreatePod) (*pod.Pod, error) {
+	if p.Image == nil {
+		return nil, gimlet.ErrorResponse{
+			StatusCode: http.StatusBadRequest,
+			Message:    fmt.Sprintf("Invalid API input: missing image"),
+		}
+	}
```

Comment:
If you're actually going to do the validation here, CPU and memory are also required, as is platform.